Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1848,9 +1852,18 @@ void CGContextSetFillPattern(CGContextRef context, CGPatternRef pattern, const C
     }
 
     ComPtr<ID2D1BitmapBrush1> bitmapBrush;
-    FAIL_FAST_IF_FAILED(_CreatePatternBrush(context, pattern, components, &bitmapBrush, [&](CGContextRef drawingContext) {
+    FAIL_FAST_IF_FAILED(_CreatePatternBrush(context, pattern, &bitmapBrush, [&](CGContextRef drawingContext, bool isColored) {
```

Comment:
no and we don't need to keep on passing it.
I just did it at one place, to add the comments on what's happening with the colored/non-colored indication.

---
In reply to: [101328139](https://github.com/Microsoft/WinObjC/pull/1952#discussion_r101328139) [](ancestors = 101328139)